gravitee-apim-console-webui/src/management/api/api-notification/components/notification-edit-dialog/notification-edit-dialog.component.scss

```diff
@@ -0,0 +1,41 @@
+@use 'sass:map';
+
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../../scss/gio-layout' as gio-layout;
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.category {
+  margin-bottom: 20px;
+
+  h6 {
+    margin: 0;
+  }
+}
+
+.events {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.event {
+  flex-basis: calc(50% - 8px);
+  margin: 4px;
+
+  &__label {
+    font-weight: bold;
```

Comment:
maybe more : `@include mat.typography-level($typography, subtitle-2);` to use the common "body strong"